addons/OPT/FIELDREPAIR/fn_vehicleDamagedLight.sqf

```diff
@@ -17,7 +17,7 @@
 params [["_veh", objNull]];
 
 if (_veh isEqualTo objNull) exitWith {false};
-if (count _veh <= 0) exitWith {false};
+if (count (getAllHitPointsDamage _veh) <= 0) exitWith {false};
```

Comment:
Ich habe es noch nicht getestet, aber macht es hier nicht mehr Sinn bei <3 abzubrechen, da wir in der nächsten Zeile ja auf das 3. Element zugreifen?

Reply:
Guter Punkt Teste ich mal.

Reply:
Getestet und geht. Fangen somit auch den Case des nur 2 großen Arrays ab.